src/gradualizer_lib.erl

```diff
@@ -109,6 +109,8 @@ pick_value(?type(tuple, any)) ->
     {};
 pick_value(?type(tuple, Tys)) ->
     list_to_tuple([pick_value(Ty) || Ty <- Tys]);
+pick_value(Rec = ?type(record, _)) ->
+    typelib:pp_type(Rec);
```

Comment:
Pretty-printing the type doesn't seem like the right thing to do here. It should return an example value of the type.

Reply:
I went for pp_type to have "pretty" record value printed out (instead of the usual tuple)

`pick_value` is only used in one place:
- https://github.com/josefs/Gradualizer/blob/3e55441b2fbc32f84636ec05b43f487203cddd4f/src/typechecker.erl#L3211
- https://github.com/josefs/Gradualizer/blob/3e55441b2fbc32f84636ec05b43f487203cddd4f/src/gradualizer_fmt.erl#L48-L54

Reply:
Would it make sense to replace the call to `io_lib:format` with some custom logic with `io_lib_pretty:print`?
That way we would have a way to preserve both machine and human consumption of the suggested value.